lib/_foreign_function.js

```diff
@@ -6,6 +6,7 @@
 var assert = require('assert')
   , debug = require('debug')('ffi:_ForeignFunction')
   , ref = require('ref')
+  , Promise = require('any-promise')
```

Comment:
We still test 0.10, and a lot of users use bluebird which has `.finally`. Our promises should match the user's, that's the point of any-promise.